### PR TITLE
Setup Very Basic Request and Add Cors to Server 

### PR DESCRIPTION
**Pull request by MsSammieRose**

### CODE REVIEW

File: reminder-app/src/store/api.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
```
Comment (remvst) [inline reminder-app/src/store/api.tsx]: I think this file can just be .ts. We won't create components here

File: server/src/index.ts [hunk 2]
```diff
@@ -33,8 +34,14 @@ connection.connect((err) => {
 const reminderDao = new ReminderDAO(connection);
 
 const app = express();
+
+
 app.use(express.json());
 
+app.use(cors({
+    origin: '*'
+  }));
```
Comment (remvst) [inline server/src/index.ts]: Extra indentation here :(

File: reminder-app/src/store/api.tsx (hunk 1 above)
Comment (MsSammieRose) [reply reminder-app/src/store/api.tsx]: Great catch! Thank you for your input

File: server/src/index.ts (hunk 2 above)
Comment (MsSammieRose) [reply server/src/index.ts:43]: fight me

Comment (MsSammieRose) [reply server/src/index.ts]: Of course! I've run prettier :D

**Review by remvst — APPROVED** [review]

**Pull request merged**